src/test/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/inference/AssemblyBasedSVDiscoveryTestDataProviderForSimpleSV.java

```diff
@@ -0,0 +1,1032 @@
+                forComplexTanDup_3to2_noPseudoHom_minus,
+                forComplexTanDup_2to3_noPseudoHom_plus,
+                forComplexTanDup_2to3_noPseudoHom_minus,
+                forComplexTanDup_1to2_short_pseudoHom_plus,
+                forComplexTanDup_1to2_short_pseudoHom_minus,
+                forComplexTanDup_2to3_short_noPseudoHom_plus,
+                forComplexTanDup_2to3_short_noPseudoHom_minus);
+        return Collections.unmodifiableList(testDataForSimpleSVs);
+    }
+
+    // same event, two representations from opposite strands
+    public static List<Tuple2<TestDataForSimpleSV, TestDataForSimpleSV>> getAllTestDataPaired() {
+        final List<TestDataForSimpleSV> allTestData = getAllTestData();
+        final List<Tuple2<TestDataForSimpleSV, TestDataForSimpleSV>> testDataForSimpleSVs
+                = new ArrayList<>(allTestData.size()/2);
+        for (int i = 0; i < allTestData.size() - 1; i += 2) {
+            testDataForSimpleSVs.add(new Tuple2<>(allTestData.get(i), allTestData.get(i+1)));
+        }
+        return Collections.unmodifiableList(testDataForSimpleSVs);
+    }
+
+    private static VariantContextBuilder addStandardAttributes(final VariantContextBuilder inputBuilder,
+                                                               final int alignLength, final String contigName,
+                                                               final String variantType, final int end,
+                                                               final int svLen, final String altSeq,
+                                                               final String homSeq, final String insSeq) {
+        VariantContextBuilder builder = inputBuilder
+                .attribute(TOTAL_MAPPINGS, 1).attribute(HQ_MAPPINGS, 1)
+                .attribute(ALIGN_LENGTHS, alignLength).attribute(MAX_ALIGN_LENGTH, alignLength)
+                .attribute(CONTIG_NAMES, contigName)
+                .attribute(VCFConstants.END_KEY, end)
+                .attribute(SVTYPE, variantType)
+                .attribute(SVLEN, svLen)
+                .attribute(MAPPING_QUALITIES, 60);
+        if (!homSeq.isEmpty()) {
+            builder.attribute(HOMOLOGY, homSeq).attribute(HOMOLOGY_LENGTH, homSeq.length());
+        }
+        if (!insSeq.isEmpty()) {
+            builder.attribute(INSERTED_SEQUENCE, insSeq).attribute(INSERTED_SEQUENCE_LENGTH, insSeq.length());
+        }
+        if (!altSeq.isEmpty()) {
+            builder.attribute(SEQ_ALT_HAPLOTYPE, altSeq);
+        }
+        return builder;
+    }
+
+    /**
+     * 40-'A' + 10-'C'+10-'T' + 40-'G' where the segment 10-'C'+10-'T' is deleted (forward strand representation description).
+     *
+     * Return a list of two entries for positive and reverse strand representations.
+     */
+    private static List<TestDataForSimpleSV>
+    forSimpleDeletion(final ByteArrayOutputStream outputStream) throws IOException {
```

Comment:
It would be nice in these tests if you could add a few comments to each one demarcating the input test data and the expected results. Otherwise it's hard to figure out where one stops and the other stops.